assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -109,10 +106,10 @@ const alignmentButtonIds = [
 const PAGE_RECT = {
   startX: 0,
   startY: 0,
-  endX: DEFAULT_EDITOR_PAGE_WIDTH,
-  endY: DEFAULT_EDITOR_PAGE_HEIGHT,
-  width: DEFAULT_EDITOR_PAGE_WIDTH,
-  height: DEFAULT_EDITOR_PAGE_HEIGHT,
+  endX: ALLOWED_EDITOR_PAGE_WIDTHS[0],
```

Comment:
Likewise, not relevant, but still: this should be `PAGE_WIDTH` and `PAGE_HEIGHT` instead here. If you're not comfortable to make this change here, please file a bug and assign it to the blame author. All the math below is happening with "data units", not with "editor units".